tests/scalers/cassandra.test.ts

```diff
@@ -152,7 +162,7 @@ spec:
         app: cassandra-app
     spec:
       containers:
-      - image: bitnami/cassandra:4.0.1
```

Comment:
I did this change, because bitnami/cassandra is non-root, so please revert this change.